### ISSUE TITLE
Not movable/deletable decals appearing on custom map.

### ISSUE THREAD

**Issue by wsmajt** [SmartlyDressedGames/Unturned-3.x-Community#4145]
Hi,
after the update i noticed that when im switching maps in editor im starting to see a non moveable/deletable decals on map.
![1](https://github.com/SmartlyDressedGames/Unturned-3.x-Community/assets/70204246/af3b8780-67d0-498e-b765-919ea8a333c9)
![2](https://github.com/SmartlyDressedGames/Unturned-3.x-Community/assets/70204246/a64c42c5-7935-49e8-a136-0e88f2842c18)

**Comment by SDGNelson** [comment]
Are they decals you placed? Or completely random? If completely random, I wonder if there is an exception being thrown that's preventing them from getting cleaned up when changing maps. In that case, could you send your Client.log file after it happens again please?

**Comment by wsmajt** [comment]
Its decals that i put on previous map + some random.
[Client.log](https://github.com/SmartlyDressedGames/Unturned-3.x-Community/files/12922849/Client.log)

**Comment by wsmajt** [comment]
i saw no exception in log regarding it.

**Comment by SDGNelson** [comment]
You're right that there don't seem to be any related exceptions. Does it happen consistently when switching between two of your custom maps? If so, I wonder if you could share the maps so I can reproduce it and figure out where the decals are coming from?

**Comment by wsmajt** [comment]
I uploaded the logs so u can see yourself. In this logs i launched game went to first map then the second.
if u want it i could release in private my files.

**Comment by wsmajt** [comment]
`[2023-10-17 19:09:38] Context "Object" missing asset 015babca05224b77b86e48ae854642d9
[2023-10-17 19:09:38] Context "Object" known missing asset 015babca05224b77b86e48ae854642d9, server will not kick clients for this
[2023-10-17 19:09:38] Unable to find object for GUID 015babca05224b77b86e48ae854642d9, nor by legacy ID 57146
[2023-10-17 19:09:38] Object with no asset in region 29, 32: 57146 015babca-0522-4b77-b86e-48ae854642d9
[2023-10-17 19:09:38] Unable to find object for GUID 015babca05224b77b86e48ae854642d9, nor by legacy ID 57146
[2023-10-17 19:09:38] Object with no asset in region 29, 32: 57146 015babca-0522-4b77-b86e-48ae854642d9
[2023-10-17 19:09:38] Unable to find object for GUID 92f8c495b6c641cea085d377c2f14cee, nor by legacy ID 57153
[2023-10-17 19:09:38] Object with no asset in region 29, 32: 57153 92f8c495-b6c6-41ce-a085-d377c2f14cee
[2023-10-17 19:09:38] Unable to find object for GUID d070491f5efc46e68a1e36d2ab1c15b3, nor by legacy ID 23147
[2023-10-17 19:09:38] Object with no asset in region 29, 32: 23147 d070491f-5efc-46e6-8a1e-36d2ab1c15b3
[2023-10-17 19:09:38] Unable to find object for GUID 373c147d4642475aa5a6c1eeb5c6db9c, nor by legacy ID 57279
[2023-10-17 19:09:38] Object with no asset in region 29, 32: 57279 373c147d-4642-475a-a5a6-c1eeb5c6db9c
[2023-10-17 19:09:38] Unable to find object for GUID 373c147d4642475aa5a6c1eeb5c6db9c, nor by legacy ID 57279
[2023-10-17 19:09:38] Object with no asset in region 29, 32: 57279 373c147d-4642-475a-a5a6-c1eeb5c6db9c
[2023-10-17 19:09:38] Unable to find object for GUID 373c147d4642475aa5a6c1eeb5c6db9c, nor by legacy ID 57279
[2023-10-17 19:09:38] Object with no asset in region 29, 32: 57279 373c147d-4642-475a-a5a6-c1eeb5c6db9c
[2023-10-17 19:09:38] Unable to find object for GUID b3f1630508ff42e782ea5785429540da, nor by legacy ID 57152
[2023-10-17 19:09:38] Object with no asset in region 29, 32: 57152 b3f16305-08ff-42e7-82ea-5785429540da
[2023-10-17 19:09:38] Unable to find object for GUID 5192a02b1c424758a8bf0c6034ad5743, nor by legacy ID 57258
[2023-10-17 19:09:38] Object with no asset in region 29, 32: 57258 5192a02b-1c42-4758-a8bf-0c6034ad5743
[2023-10-17 19:09:38] Unable to find object for GUID 061fb5dd9870457b87c008e62b22066d, nor by legacy ID 44174
[2023-10-17 19:09:38] Object with no asset in region 29, 32: 44174 061fb5dd-9870-457b-87c0-08e62b22066d
[2023-10-17 19:09:38] Unable to find object for GUID 9a288a60f9594e33a83d95001758c319, nor by legacy ID 57149
[2023-10-17 19:09:38] Object with no asset in region 29, 32: 57149 9a288a60-f959-4e33-a83d-95001758c319
[2023-10-17 19:09:38] Unable to find object for GUID e60242956f5d45b9899b82ca281f0ec5, nor by legacy ID 57033
[2023-10-17 19:09:38] Object with no asset in region 29, 32: 57033 e6024295-6f5d-45b9-899b-82ca281f0ec5
[2023-10-17 19:09:38] Unable to find object for GUID e60242956f5d45b9899b82ca281f0ec5, nor by legacy ID 57033
[2023-10-17 19:09:38] Object with no asset in region 29, 32: 57033 e6024295-6f5d-45b9-899b-82ca281f0ec5`
i found it in log files but i dont know if its anything related

**Comment by wsmajt** [comment]
i checked with map that im not the author and it happened again.
![3](https://github.com/SmartlyDressedGames/Unturned-3.x-Community/assets/70204246/48a29cf5-47d5-4d24-8ef7-a4bbffb2f66a)

**Comment by wsmajt** [comment]
Its happening even with vanilla decals :
![4](https://github.com/SmartlyDressedGames/Unturned-3.x-Community/assets/70204246/33b0072c-60b9-473b-acb2-ee3c109886a8)
![5](https://github.com/SmartlyDressedGames/Unturned-3.x-Community/assets/70204246/8212b0fe-c661-45b9-8b2f-bef1bbf98a5b)

**Comment by SDGNelson** [comment]
Okay, sorry for my delay, I've accepted your invite on Discord to share the files.

**Comment by SDGNelson** [comment]
Thank you for the example maps! I was able to reproduce it using your maps and mods, and it should be fixed for the next update.
